source/about/glossary.rst

```diff
@@ -25,6 +25,9 @@ Crowdin
   https://crowdin.com/project/sponge-docs
 Developer
   Someone who programs code (Sponge, Plugins, Mods, ...).
+Discord
+  Discord is a free voice, video and text chat app, and is used as a chat platform. Like IRC, Provides chat rooms for all things Sponge.
```

Comment:
Please try to keep lines under 120 characters.

Reply:
Will do!